--- ./src/Makefile.org	Fri Oct  5 19:04:52 2001
+++ ./src/Makefile	Fri Oct  5 19:05:21 2001
@@ -220,9 +220,9 @@
 
 install: $(TARGET) $(MAN_FILE).gz
 	mkdir -p $(INSTALL_MOD_PATH)$(INSTDIR)
-	install -m 644 -o root $(TARGET) $(INSTALL_MOD_PATH)$(INSTDIR)
+	install -m 644 $(TARGET) $(INSTALL_MOD_PATH)$(INSTDIR)
 	mkdir -p $(INSTALL_MOD_PATH)$(MAN_INSTDIR)
-	install -m 644 -o root $(MAN_FILE).gz $(INSTALL_MOD_PATH)$(MAN_INSTDIR)
+	install -m 644 $(MAN_FILE).gz $(INSTALL_MOD_PATH)$(MAN_INSTDIR)
 ifeq (,$(INSTALL_MOD_PATH))
 	/sbin/depmod -a || true
 else
