--- e100-2.1.15/src/e100_main.c~	Wed Mar  3 23:22:14 2004
+++ e100-2.1.15/src/e100_main.c	Wed Mar  3 23:22:14 2004
@@ -87,6 +87,7 @@
 #undef __NO_VERSION__
 
 #include <linux/config.h>
+#include <asm/current.h>
 #include <net/checksum.h>
 #include <linux/tcp.h>
 #include <linux/udp.h>
--- e100-2.1.15/src/e100.h~	Wed Mar  3 23:28:06 2004
+++ e100-2.1.15/src/e100.h	Wed Mar  3 23:28:06 2004
@@ -28,6 +28,7 @@
 #ifndef _E100_INC_
 #define _E100_INC_
 
+#include <asm/current.h>
 #include <linux/module.h>
 #include <linux/types.h>
 #include <linux/init.h>
--- e100-2.1.15/src/e100_kcompat.h~	Wed Mar  3 23:41:04 2004
+++ e100-2.1.15/src/e100_kcompat.h	Wed Mar  3 23:41:04 2004
@@ -38,6 +38,7 @@
 #define __E100_KCOMPAT_H__
 
 #include <linux/version.h>
+#include <asm/current.h>
 
 /******************************************************************
  *#################################################################
